src/Nethermind/Nethermind.Merge.Plugin/Data/ExecutionPayloadV3.cs

```diff
@@ -35,6 +37,14 @@ public ExecutionPayloadV3(Block block) : base(block)
     /// </summary>
     public ulong? ExcessDataGas { get; set; }
 
+
+    /// <summary>
+    /// Gets or sets <see cref="Block.ParentBeaconBlockRoot"/> as defined in
+    /// <see href="https://eips.ethereum.org/EIPS/eip-4788">EIP-4788</see>.
+    /// </summary>
+    [JsonIgnore]
```

Comment:
really hoping they agree to including it in the payload instead of the call arg.

Reply:
same, it's ugly and forces us to do work arounds to not have the ParentBeaconBlockRoot argument creep out of control